script/Arcueid/Day 11/11_00_ARC11_3C.txt

```diff
@@ -513,7 +513,7 @@ Not even tears will come.
 [sha:bbdaed93d535b175d10a99d37dc187b9b963f411]{
 -- Page 322, Offset 30468.
 -- ぬけがらのまま、足が動く。
-Despite being hollow, my legs begin to move.
+In my hollowness, my legs begin to move.
```

Comment:
This doesn't really read right to me. I'd either revert or maybe
> My lifeless legs begin to move.
> Devoid of everything, my legs begin to move.

Reply:
if you want to change to one of those, the latter is closer to the nuance Petrikow wanted.